specs/elements/Input.Toggle.md

```diff
@@ -97,17 +84,17 @@ Controls the amount of spacing between this element and the preceding element.
 ## Rendering
 
 ### Labels
-The `label` property should be rendered to the left of the toggle box. Clicking/tapping on the rendered label should toggle the input.
+The `label` property should be rendered above the input box if present. Clicking/tapping on the rendered label should put focus in the input box. 
+
+The `color`, `isSubtle`, `size`, and `weight` properties provided in the host config under `"inputs":"label":"requiredInputs"` for inputs with `"isRequired":true`, and under `"inputs":"label":"optionalInputs"` for inputs with `"isRequired":false` should be respected. The spacing between the label and the input should be set to the value of `inputSpacing` specified in those same locations in host config.
 
 ### Accessibility
 The `label` property should be set as the accessibility text when present. If the `label` property is not present, the `placeholder` property should be used instead if present.
```

Comment:
What precisely do we imply by "set as the Accessibility text" ? What if the element has both a placeholder as well as a title/label defined since ideally placeholder wouldn't be use as an alternative to label ?

Reply:
This is a good question that I don't know the answer to.  @almedina-ms do you know what the expected behavior is here, or how it behaves today on the implemented platforms?

---
In reply to: [449977721](https://github.com/microsoft/AdaptiveCards/pull/4255#discussion_r449977721) [](ancestors = 449977721)

Reply:
When I wrote this I meant "accessibility text" as the text that is read by narrator but since I have changed my view in the matter, I don't think placeholder should be used as something the can be a replacement for labels as they don't behave the same and it's not recommended by the accessibility teams. Only label should be read by narrator, while we should make card authors aware of using placeholders as labels is bad practice

Reply:
What is the best practice for exposing placeholder text to narrator in the case where it's not being used as a label? Presumably whatever is in there (a hint, or whatever) would be useful to narrator users as well, right? Does the accessibility team have a recommendation there? 

Also, if using placeholders as labels is bad practice, we probably shouldn't have our our main input scenario card sample card doing that :) (See [Inputs.json](https://github.com/microsoft/AdaptiveCards/blob/main/samples/v1.0/Scenarios/Inputs.json)). I can log a separate issue to update it if that makes sense to people. (cc @matthidinger)

![image](https://user-images.githubusercontent.com/9091833/87188198-40bff080-c2a3-11ea-8579-e61f2e4a5244.png)